h/groups/__init__.py

```diff
@@ -1,4 +1,8 @@
 # -*- coding: utf-8 -*-
 
+from h.groups import search
+
+
 def includeme(config):
     config.memex_set_groupfinder('h.groups.util.fetch_group')
+    config.memex_add_search_filter(search.GroupAuthFilter)
```

Comment:
Also, to avoid more circular import dependencies, it might be nice to add a `config.maybe_dotted` to `memex_add_search_{filter,matcher}` while you're here, so you can get rid of the import in this module.